lcoe_calculator_documentation.html

```diff
@@ -139,8 +139,9 @@ <h2>Data Sources</h2>
         </div>
       </p>
       <h2>Source code</h2>
-      <p>The calculator is powered by a JavaScript file that is <a href='js/lcoe_calculator.js'>available online</a>.</p>
+      <p>The source code for this calculator is publicly available <a href='https://github.com/NREL/PVLCOE'>on GitHub</a>.</p>
       <h2>Changelog</h2>
+      <p><strong>2020-09-11: Changed source code link to GitHub repository; added explanation of system versus module degradation rates.</strong></p>
```

Comment:
I suggest adding a note here about the addition of the xlsx and the associated notebook

Reply:
Thanks -- done!